sycl/plugins/cuda/pi_cuda.cpp

```diff
@@ -2565,12 +2581,8 @@ pi_result cuda_piEnqueueKernelLaunch(
   pi_uint32 local_size = kernel->get_local_size();
 
   {
-    pi_result retError = cuda_piKernelGetGroupInfo(
-        kernel, command_queue->device_,
-        PI_KERNEL_GROUP_INFO_COMPILE_WORK_GROUP_SIZE,
-        sizeof(reqdThreadsPerBlock), reqdThreadsPerBlock, nullptr);
-    assert(retError == PI_SUCCESS);
-
+    kernel->get_reqd_threads_per_block(sizeof(reqdThreadsPerBlock),
+                                       reqdThreadsPerBlock);
```

Comment:
Very minor, but instead of having `get_reqd_threads_per_block` and `reqdThreadsPerBlock` in this scope, you could just use `kernel->reqdThreadsPerBlock_` directly. Avoids redundant memory and `reqdThreadsPerBlock_` should only be set once.

Reply:
done. thank you for the comment!